text-notification/index.js

```diff
@@ -0,0 +1,85 @@
+module.exports = class TextNotificationPlugin extends BasePlugin {
+
+    /** Plugin info */
+    static get id()             { return 'zongo.text-notify' }
+    static get name()           { return 'Notify Hosts' }
```

Comment:
Users may get confused when installing this plugin, as they think they are installing the "Text Notification" plugin but instead it will show up as "Notify Hosts"